templates/openstack/features.html

```diff
@@ -246,12 +242,33 @@ <h3>Kubernetes support included</h3>
     <div class="row">
```

Comment:
I don't like this section's look

can we try this?

```
<section class="p-strip is-bordered">
    <div class="row">
      <h2>Storage</h2>
    </div>
        <div class="row">
        <div class="col-6">
        <p>Storage performance and economics are tricky to balance in a cloud environment. Canonical will help you architect your storage across the cluster to balance price and performance, ensuring the right mix of resilience, latency, IOPS and integrity for your particular deployment.</p>
        <ul class="p-list">
          <li class="p-list__item is-ticked">12TB usable storage per node included</li>
          <li class="p-list__item is-ticked">Mix of local direct-attach and network storage</li>
          <li class="p-list__item is-ticked">Distributed software-defined storage vs SAN assessment</li>
          <li class="p-list__item is-ticked">Performance tuning for local and network storage</li>
          <li class="p-list__item is-ticked">Understanding storage compression</li>
          <li class="p-list__item is-ticked">Understanding storage resilience</li>
          <li class="p-list__item is-ticked">Third-party storage solutions available for integration</li>
          <li class="p-list__item is-ticked">Object stores and other higher-level storage constructs</li>
          <li class="p-list__item is-ticked">Hierarchical storage configurations</li>
        </ul>
        <p><a href="/openstack/storage" class="">Learn more about storage with Ubuntu Advantage for Infrastructure ›</a></p>
    </div>
        <div class="col-6">
        <div class="p-heading-icon">
          <div class="p-heading-icon__header">
            <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}4f1e16bf-image-swift.svg">
            <h3 class="p-heading-icon__title">Swift</h3>
          </div>
          <p>The high performance S3-compatible object store developed and shipped with OpenStack, ideal for storing unstructured data.</p>
        </div>
          <div class="p-heading-icon">
          <div class="p-heading-icon__header">
            <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}636b94fd-ceph.svg">
            <h3 class="p-heading-icon__title">Ceph</h3>
          </div>
          <p>A converged storage framework that has been designed to present object, block, and file storage from a single distributed computer cluster.</p>
        </div>
      </div>
  </div>
</section>
```

![image](https://user-images.githubusercontent.com/441217/56839605-2a36a080-687b-11e9-93ec-c601122a6ed4.png)